puppet/site.pp

```diff
@@ -1,75 +1,87 @@
 # Load modules and classes
-hiera_include('classes')
+lookup('classes', {merge => unique}).include
 
 $solr_path = "/opt/solr"
 $solr_tools_path = "/opt/search-solr-tools"
 
-class update-package-manager {
-  exec { "update":
-    path => "/bin:/usr/bin:/usr/local/bin:/usr/local/sbin:usr/sbin:/sbin:/usr/java/jdk/bin",
-    command => "apt-get -y update; sudo apt-get -y install libxml2 libxml2-dev libxslt1-dev"
-  }
-  notify { "apt-get update complete":
-    require => Exec['update']
-  }
-}
+# class update_package_manager {
+#   exec { "update":
+#     path => "/bin:/usr/bin:/usr/local/bin:/usr/local/sbin:usr/sbin:/sbin:/usr/java/jdk/bin",
+#     command => "apt-get -y update; sudo apt-get -y install libxml2 libxml2-dev libxslt1-dev"
+#   }
+#   notify { "apt-get update complete":
+#     require => Exec['update']
+#   }
+# }
 
 ### BEGIN nokogiri deps
-Class['update-package-manager'] -> Package <| |>
+# Class['update_package_manager'] -> Package <| |>
 
-package {"libssl-dev":
+exec { 'apt-get update':
```

Comment:
This shouldn't be necessary, since the `apt-get update` step happens before the puppet provisioner is run (in the Vagrantfile)